src/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -101,7 +101,7 @@ public enum ProtectionLevel
     public delegate bool RemoteCertificateValidationCallback(object sender, System.Security.Cryptography.X509Certificates.X509Certificate certificate, System.Security.Cryptography.X509Certificates.X509Chain chain, System.Net.Security.SslPolicyErrors sslPolicyErrors);
     public class SslServerAuthenticationOptions
     {
-        public bool AllowRenegotiation { get { throw null;  } set { } }
+        public bool DisableRenegotiation { get { throw null;  } set { } }
```

Comment:
Personally I prefer the Allow with a default of true over having to understand the double negative of  Disable with a default of false.

Was this change reviewed by API review?

Reply:
I didn't take it through API-review, since bool default value is false, I thought the negation made more sense, atleast in my head. Can revert it to Allow.